### ISSUE TITLE
Add Ubuntu 16.04 support

### ISSUE THREAD

**Issue by arm4b** [StackStorm/chef-stackstorm#19]
`Ubuntu 16.04` packages are currently in `unstable` quality and available under the [staging](https://packagecloud.io/StackStorm/staging-unstable) repos. 

TODO: Include & Verify Ubuntu Xenial support within cookbook when StackStorm ships the corresponding packages. The change is probably just about 1 line in `kitchen.yml`

> P.S. It's just a reminder, not to forget about it when we release U16 packages

**Comment by shortdudey123** [comment]
https://github.com/StackStorm/chef-stackstorm/blob/master/recipes/_packages.rb#L22 can be modified to use the staging repo when the system is ubuntu 16.04+

Or are you more thinking adding it to kitchen testing when a stable version is released?

**Comment by arm4b** [comment]
Yeah, adding `xenial` support after shipping StackStorm packages in `stable` packagecloud repo would be more consistent.

**Comment by shortdudey123** [comment]
:+1:

**Comment by shortdudey123** [comment]
:+1:

**Comment by docwhat** [comment]
Ubuntu 14.04 has its _End of Life_ coming up in April of 2019 (a couple weeks away)...

It also looks like packages are available for Xenial/16.04 in [stable](https://packagecloud.io/app/StackStorm/stable/search?dist=ubuntu%2Fxenial).

**Comment by arm4b** [comment]
@docwhat Indeed. I think we’ll still support U14 for a while until stopping shipping deb packages after end of Trusty life.
If you or someone wants to add Xenial support for this cookbook, - it should be fairly straightforward to add and we’re absolutely open to any additions, as this repo is mainly community-contributed.
